lib/gitarro/backend.rb

```diff
@@ -4,8 +4,33 @@
 require 'optparse'
 require 'time'
 require 'English'
+require 'faraday-http-cache'
 require_relative 'opt_parser'
 require_relative 'git_op'
+require 'active_support'
+
+# by default enabled (faraday_cache in memory, will be deleted
+# after gitarro run 2nd time. usefull for -C check option
+# and performance, since we cache)
+module CachingOctokit
+  def create_dir_store(cache_path)
+    cache_name = '/gitarro/httpcache'
```

Comment:
IMHO this should be:
```cache_name = 'httpcache'````
 
And then ```cache_path``` path would be ```/tmp/gitarro``` by default or the value the user wants to use.

This way it's possible to tell gitarro to store it's cache at ~./gitarro/httpcache or whatever we want.

In the end, the idea for the future is to tell gitarro the base directory for its data.

Reply:
ok